pkg/manifests/testdata/artifacts/blobs-only/blobs/sha256/579b4ca613dbc3017ec77e006dc9a275e074591d1c19b635e49aaf2dd07cfe3d

```diff
@@ -0,0 +1,17 @@
+{
+  "schemaVersion": 2,
+  "mediaType": "application/vnd.oci.image.manifest.v1+json",
+  "artifactType": "application/vnd.example+type",
+  "config": {
+    "mediaType": "application/vnd.example.config.v1+json",
```

Comment:
My own sloppiness aside, the config field isn't optional, so we have to include something here, and while layers is a slice, I don't think an empty slice is allowed.
If I'm reading https://github.com/opencontainers/image-spec/blob/main/manifest.md#guidelines-for-artifact-usage correctly, the `DescriptorEmptyJSON` value is the recommended value for fields that wouldn't be there otherwise.